### PR TITLE
Initial BOSH release for our Consul cluster

### PR DESCRIPTION
**Pull request by vcardillo**
https://www.pivotaltracker.com/story/show/150426432

- [x] 3 instances
- [x] Multi-AZ
- [x] TLS
- [x] Encrypted EBS volumes

The UI:
https://172.16.31.6:8500/ui/#/dc1/services/consul

### CODE REVIEW

**Comment by jeremiemv** [comment]
Solid checklist.

File: README.md [hunk 1]
```diff
@@ -19,9 +19,12 @@ cd consul-boshrelease
 bosh deploy manifests/consul.yml -o manifests/operators/firsttime.yml
 ```
 
-If your BOSH does not have Credhub/Config Server, then remember ` --vars-store` to allow generation of certificates.
+If your BOSH does not have Credhub/Config Server, then remember ` --vars-store` to allow generation of certificates. Such as:
+
+```
+bosh -e tools -d consul deploy ./manifests/consul.yml -o manifests/operators/firsttime.yml --vars-store ./creds.yml
```
Comment (papapoison) [inline README.md]: Love this addition

File: manifests/consul.yml [hunk 2]
```diff
@@ -5,12 +5,12 @@ name: consul
 
 instance_groups:
 - name: consul
-  azs: [z1, z2, z3]
+  azs: [z1, z3, z4]
```
Comment (papapoison) [inline manifests/consul.yml]: Where are these defined?

**Comment by vcardillo** [comment]
@adamstegman If the manifest can go in the deployments repo, then there's no point to this repository.

**Comment by vcardillo** [comment]
Closing. See: https://github.com/onemedical/deployments/pull/284/files

**Pull request closed without merging**